content/en-US/api/isExtended.md

```diff
@@ -0,0 +1,35 @@
+---
+recipe: api-interface
+title: 'Screen'
+mdn_url: /en-US/docs/Web/API/Screen/isExtended
+specifications: https://webscreens.github.io/window-placement/#dom-screen-isextended
+browser_compatibility: api.Screen.isExtended
+---
+
+## Description
+
+The `isExtended` read-only property of the `Screen` interface returns true if the device has multiple screens that can be used for window placement.
+
```

Comment:
This is an example of something I'm not going to put in the templates. I'll just add it when it comes up.

Reply:
Did this suggestion end up in the wrong file somehow?

Reply:
Oh, I think I see the confusion. This property intentionally in Screen not ScreenAdvanced; knowing about ScreenAdvanced is irrelevant for this property. It will end up documented with all of the other properties of Screen.

Reply:
This did end up in the wrong file. It belongs in the interface. Members use the idiom "Inherits properties/methods from..."